### PR TITLE
solution

### PR DESCRIPTION
**Pull request by igor-kashchenko**

### CODE REVIEW

File: src/makeInfinityAdder.js [hunk 1]
```diff
@@ -6,7 +6,21 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  return function adder(num) {
+    if (num !== 0 && !num) {
```
Comment (vanst1) [inline src/makeInfinityAdder.js:12]: you can use `num === undefined`

**Review by stadnytskyi-ivan — APPROVED** [review]
Well done;)

File: src/makeInfinityAdder.js (hunk 1 above)
Comment (alexandra-protyanova) [reply src/makeInfinityAdder.js:12]: agree

File: src/makeInfinityAdder.js [hunk 2]
```diff
@@ -6,7 +6,21 @@
...
+      const sums = sum;
```
Comment (alexandra-protyanova) [inline src/makeInfinityAdder.js:13]: bad name for variable

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]

**Review by alexandra-protyanova — APPROVED** [review]

**Pull request closed without merging**